frontend/src/components/borrow/MyLoanTile.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="my-loan-tile">
+    <div class="mlt-details">
+        <div class="mlt-short-desc">{{ loan.description }}</div>
+    </div>
+    <div class="mlt-financial-data">
+      <div>
+        <div class="mlt-fin-line-a"> AMOUNT BORROWED </div>
+        <div class="mlt-fin-line-b">{{ loan.amountWanted }} {{ loan.tokenSymbol }}</div>
+      </div>
+      <div>
+        <div class="mlt-fin-line-a"> RATE </div>
+        <div class="mlt-fin-line-b">{{ loan.interestPermil }}%</div>
+        <!-- div class="mlt-fin-line-c">{{ loan.apr }} APR</div -->
```

Comment:
Don't commit commented out code.

Reply:
Then add a comment about why there is commented out code.

Otherwise, I do think that whoever will implement this in the future will be able to find out how to use this class :).